test/openstreetmap-polygons.js

```diff
@@ -30,7 +30,7 @@ describe('openstreetmap-polygons', function () {
     _extractor.start(function (err, results) {
       _error = err;
       _results  = results;
-      done();
+      setTimeout(done, 30);
```

Comment:
what does this 30ms refer to? is that the time it takes to spawn a child process?